janus_core/helpers/mlip_calculators.py

```diff
@@ -305,6 +305,27 @@ def choose_calculator(
             potential=potential, load_path=model_path, device=device, **kwargs
         )
 
+    elif arch == "fairchem":
+        from fairchem.core import OCPCalculator, __version__
+
+        # Default model
+        model_path = model_path if model_path else "EquiformerV2-31M-S2EF-OC20-All+MD"
+
+        model_name = None
+        checkpoint_path = None
+
+        if isinstance(model_path, Path) and model_path.exists():
+            checkpoint_path = str(model_path)
+        else:
+            model_name = str(model_path)
+
+        kwargs.setdefault("local_cache", "pretrained_models")
```

Comment:
Shall we change this to be somewhere a bit more hidden? Maybe `~/.cache/fairchem`?

Reply:
you mean the default? yes but be sure it works in the sense creates the folder hierarchy